src/app/technical-record/technical-record.constants.ts

```diff
@@ -0,0 +1,157 @@
+export const AXLENUMOPTIONS = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+export const BOOLEANRADIOOPTIONS = { ['Yes']: true, ['No']: false };
+
+export const FUNCTIONOPTIONS = { ['Articulated']: 'a', ['Rigid']: 'r' };
+
+export const FUELPROPULSION = [
+  'DieselPetrol',
+  'Hybrid',
+  'Electric',
+  'CNG',
+  'Fuel cell',
+  'LNG',
+  'Other'
+];

```

Comment:
All constants and Enums have an underscore naming convention. Same pattern should be followed here

Reply:
yes thanks. to be done in 10169